src/docs/product/integrations/integration-platform/ui-components/index.mdx

```diff
@@ -27,6 +27,11 @@ The UI components are specified in the schema section of the integration details
 }
 ```
 
+### URI Guidelines
+
+The `uri` keys in your schema elements describe the URI path that will receive a request from that UI Component. Sentry will append the path in the `uri` field onto the host of your integration's **Webhook URL**.
+For example, if the Webhook URL is `https://example.com/webhook/`, and your schema's `uri` attribute is `/alerts/`, then the integration will receive requests to `https://example.com/alerts/`.
```

Comment:
Since you're not saying _if the text in the "Webhook URL" field is..._ you don't need to capitalize "webhook". It's just a general concept here and can be lowercase. Same as above.

```suggestion
For example, if the webhook URL is `https://example.com/webhook/`, and your schema's `uri` attribute is `/alerts/`, then the integration will receive requests to `https://example.com/alerts/`.
```